src/storage/page_buffer.c

```diff
@@ -234,6 +234,8 @@ typedef enum
 /* flag for asynchronous flush request */
 #define PGBUF_BCB_ASYNC_FLUSH_REQ           ((int) 0x02000000)
 
+#define PGBUF_BCB_FLUSH_NOT_NEEDED	    ((int) 0x01000000)
```

Comment:
There seems to be a naming convention in the above flags ending in `_FLAG`. Broken by the last two ones. :)

Reply:
Added `_FLAG`.